bundles/org.openhab.binding.magentatv/README.md

```diff
@@ -0,0 +1,257 @@
+# MagentaTV Binding
+
+This Binding allows controlling the Deutsche Telekom Magenta TV Media Receiver series MR4xx and MR2xx (Telekom NGTV / Huawei Envision platform). 
+The Binding does NOT support MR3xx/1xx (old Entertain system based on Microsoft technology)!
+
+Media Receivers are automatically discovered.
+You could send keys as you press them on the remote and the binding receives program information when the channel is switched (no
+The binding provides device discovery, sending keys for the remote and also receiving program information/events.
+
+## Supported Things
+
+|Thing    |Type                                                                    |
+|---------|----------------------------------------------------------------------- |
+|receiver |A MagentaTV Receiver, the binding supports multiple models (see above). |
+
+### Supported Models
+
+* Deutsche Telekom Media Receiver MR401B - fully supported
+* Deutsche Telekom Media Receiver MR201  - fully supported
+* Deutsche Telekom Media Receiver MR400  - supported with minor restrictions (POWER key)
+* Deutsche Telekom Media Receiver MR200  - supported with minor restrictions (POWER key)
+* Deutsche Telekom Media Receiver MR601  - should be supported (not verified)
+* Deutsche Telekom Media Receiver MR3xx  - NOT supported (different platform)
+* Deutsche Telekom Media Receiver MR1xx  - NOT supported (different platform)
+
+## Auto Discovery
+
+UPnP will be used to discover receivers on the local network and discover the nessesary parameters.
+The receiver needs to be powered on to get discovered.
+
+Once the receiver is discovered it can be added from the Inbox.
+Make sure to set the parameter `userId` in the Thing configuration after adding the new thing, see section Thing Configuration.
+
+Note:
+The binding uses the network settings in openHAB's system configuration to determine the local IP address.
+The device can't be discovered if the openHAB system and receiver are not on the same network (IP/Netmask).
+In this case you need to add the Thing manually or use textual configuration (.things).
+
+If you are running openHAB in a Docker container you need to make sure that UPnP discovery is available and network interfaces 
+
+## Generating the UID (`userId`)
```

Comment:
It is not being "generated", but rather retrieved.

Wouldn't it make sense to always just use one name for it? You are now mixing UID and userId everywhere. I think it would be easier for users, if you'd simply talk about the `userId` parameter throughout the documentation.

This section must not be in the "auto discovery" section, but rather in the "Thing configuration" below.

Reply:
done